manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```diff
@@ -327,11 +327,16 @@ data:
     #
     # owners: (@openshift/team-ocs-committers)
     #
-    # job:kube_pv:count is the total number of Persistent Volumes present in OCP cluster.
+    # job:kube_pv:count is the total number of Persistent Volumes created by ODF present in OCP cluster.
```

Comment:
can you add a comment explaining that the metric is deprecated and replcaed by job:odf_system_pvs:count (also indicating from which version).